components/analytics-tools/org.wso2.carbon.analytics.tools.backup/src/main/java/org/wso2/carbon/analytics/tools/backup/AnalyticsDataBackupTool.java

```diff
@@ -65,12 +65,13 @@
  */
 public class AnalyticsDataBackupTool {
 
-    private static final String TIME_PATTERN = "yy-mm-dd hh:mm:ss";
+    private static final String TIME_PATTERN = "yy-mm-dd-hh:mm:ss";
```

Comment:
When the input gets parsed, this time format gets split into two parts ( date and time) therefore getting only the date without the time. To avoid that we needed to remove the space in between.